documentation/source/python_entry_point.rst

```diff
@@ -0,0 +1,41 @@
+Python Entry Point
+==================
+
+By default, cocotb starts the :class:`~cocotb.regression.RegressionManager` as the testing framework for cocotb.
```

Comment:
This isn't hyperlinked because ``RegressionManager`` is not in the Library Reference chapter - would it make sense to add it?

Reply:
I don't know if any of the regression manager was meant to be touched by the user and documenting it makes changing it even harder. Should I remove the ref instead?